app/design/frontend/yemelyanov/eag_theme/Magento_Newsletter/templates/subscribe.phtml

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
+
+/** @var \Magento\Newsletter\Block\Subscribe $block */
+
+?>
+<div class="block newsletter">
+    <div class="title"><strong><?= $block->escapeHtml(__('Newsletter')) ?></strong></div>
+    <div class="content">
+
```

Comment:
I can see only empty line was added to this block. What about child block or text "Subscribe to our news"?
I see the placeholder label of input element was changed, but I expected to see some additional text as a new element above or somewhere in this template